django/contrib/admin/static/admin/css/base.css

```diff
@@ -623,7 +623,8 @@ td ul.errorlist li {
     padding-left: 0;
 }
 
-.errors input, .errors select, .errors textarea {
+.errors input, .errors select, .errors textarea,
+td ul.errorlist + input, select, textarea {
```

Comment:
@hramezani Can you verify these selectors please? 

Aren't the last two `, select, textarea` unscoped?

Reply:
Ah!, Thanks @carltongibson .
I've changed the selector and also changed the test to check before/after clicking on the save button. Also, I've used `ccs selector` in the test to verify the above selectors.

Reply:
Super. Sounds good. I shall take another look.